### PR TITLE
Add relevant XKCD image

### PR DESCRIPTION
**Pull request by gdm85**
Cc @GeertJohan @aleksen

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,8 @@
 # lspb
 > Leisure Specials Protocol Buffers
 
+![the general problem](https://imgs.xkcd.com/comics/the_general_problem.png I find that when someone's taking time to do something right in the present, they're a perfectionist with no ability to prioritize, whereas when someone took time to do something right in the past, they're a master artisan of great foresight.)
```
Comment (aleksen) [inline README.md]: Image is broken.

This is not good enough! Let's keep quality in mind!

**Comment by aleksen** [comment]
Good. Image fixed. Approved.

**Comment by gdm85** [comment]
Yes I added also the tooltip text, much context.

**Comment by gdm85** [comment]
@cdesousa do you approve also?

**Comment by GeertJohan** [comment]
LGTM

**Comment by cdesousa** [comment]
LGTM

**Pull request merged**